src/embind/embind.js

```diff
@@ -50,10 +50,10 @@ var LibraryEmbind = {
     Module['flushPendingDeletes'] = flushPendingDeletes;
     Module['setDelayFunction'] = setDelayFunction;
 #if IN_TEST_HARNESS
-#if NO_DYNAMIC_EXECUTION
+#if DYNAMIC_EXECUTION == 0
     // Without dynamic execution, dynamically created functions will have no
     // names. This lets the test suite know that.
-    Module['NO_DYNAMIC_EXECUTION'] = true;
+    Module['DYNAMIC_EXECUTION'] = false;
```

Comment:
How about flipping the `#if`, and then we can set it to true here? Checking `Module['DYNAMIC_EXECUTION']` seems clearer than checking `Module['DYNAMIC_EXECUTION'] === false` later down.

Reply:
Done.